doc/getting-started/installation.rst

```diff
@@ -47,6 +47,26 @@ To install this package with ``conda`` run::
 
     conda install -c conda-forge pyvista
 
+Installing the Current Development Branch from GitHub 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There may be features or bug-fixes that have been implemented in PyVista but
+have not made their way into a release.  To install ``pyvista`` from the latest
+up-to-date development branch from github, use one of the following
+
+.. code::
+
+   pip install -U git+https://github.com/pyvista/pyvista.git@main
+
+Alternatively, you can clone the repository with git and install it with pip.
+
+.. code::
+
+   git clone https://github.com/pyvista/pyvista.git
+   cd pyvista
+   pip install . -e
+
+Note the development flag ``-e``.  This allows you to change pyvista in-place without
```

Comment:
Oh! I did not know that. Thanks.

Reply:
"e" as in editable :) It only works if there's no compiled code. If we were to add compiled code for acceleration (as briefly discussed a while back on a performance issue) we'd have to rebuild at least when compiled parts are touched.